README.md

```diff
@@ -85,7 +85,7 @@ The default `trix.css` file includes styles for basic formatted content—includ
 
 Trix automatically accepts files dragged or pasted into an editor and inserts them as attachments in the document. Each attachment is considered _pending_ until you store it remotely and provide Trix with a permanent URL.
 
-To store attachments, listen for the `trix-attachment-add` event. Upload the attached files with XMLHttpRequest yourself and set the attachment’s URL attribute upon completion. See the [attachment example](https://trix-editor.org/js/attachments.js) for detailed information.
+To store attachments, listen for the `trix-attachment-add` event. Upload the attached files with XMLHttpRequest yourself and set the attachment’s URL and HREF attributes upon completion. See the [attachment example](https://trix-editor.org/js/attachments.js) for detailed information.
```

Comment:
Perhaps we should clarify that the HREF attribute is optional. And HREF looks off to me in all caps so I think we should code format them if we're going to document both: `url` and `href`.

Reply:
Yes I agree about the lower case and code formatting, also about clarifying that the HREF attribute is optional. Although I think 99% of time people will naturally want to set the `href` attribute... except for when the attachments are image files 🤔

Reply:
Yeah, maybe file attachments should set `href` to `url` unless it's set explicitly.

Reply:
Yeah, it would still be good to have an option to not set the `href`. For instance, I am not setting the `href` for image files, because I don't want them to download whenever people click them.